File: Components/WinForms/CS/GaugesMainDemo/Modules/About.cs
using System.Drawing;
using DevExpress.Tutorials;

namespace DevExpress.XtraGauges.Demos {
    public partial class About : ucOverviewPage {
        public About() {
            InitializeComponent();
        }
        protected override string Line1Text {
            get { return "The XtraGauge Suite"; }
        }
        protected override string Line2Text {
            get { return "Dashboard and analytics controls for the WinForms platform."; }
        }
        protected override string Line3Text {
            get { return "DevExpress Desktop Controls"; }
        }
        protected override string Line4Text {
            get { return votedVSM; }
        }
    }
}

File: Components/WinForms/VB/GaugesMainDemo/Modules/About.vb
Imports DevExpress.Tutorials
Imports System.Drawing
Imports Microsoft.VisualBasic


Namespace DevExpress.XtraGauges.Demos
    Partial Public Class About
        Inherits ucOverviewPage
        Public Sub New()
            InitializeComponent()
        End Sub
        Protected Overrides ReadOnly Property Line1Text As String
            Get
                Return "The XtraGauge Suite"
            End Get
        End Property
        Protected Overrides ReadOnly Property Line2Text As String
            Get
                Return "Dashboard and analytics controls for the WinForms platform."
            End Get
        End Property
        Protected Overrides ReadOnly Property Line3Text As String
            Get
                Return "DevExpress Desktop Controls"
            End Get
        End Property
        Protected Overrides ReadOnly Property Line4Text As String
            Get
                Return votedVSM
            End Get
        End Property
    End Class
End Namespace

